# Allow build on kFreeBSD.

diff -Naur mlt-0.7.2.orig/configure mlt-0.7.2/configure
--- mlt-0.7.2.orig/configure	2011-05-02 07:59:12.000000000 +0200
+++ mlt-0.7.2/configure	2011-05-13 21:56:03.000000000 +0200
@@ -82,7 +82,7 @@
 		echo "SHFLAGS=-dynamiclib"
 		echo "LDFLAGS+=`sdl-config --libs`"
 		;;
-		Linux)
+		Linux|GNU/kFreeBSD)
 		[ "$(uname -m)" = "x86_64" ] && echo "ARCH_X86_64=1" && echo "CFLAGS+=-DARCH_X86_64"
 		[ "$optimisations" = "true" ] &&
 			echo "OPTIMISATIONS+=-ffast-math"
diff -Naur mlt-0.7.2.orig/src/mlt++/configure mlt-0.7.2/src/mlt++/configure
--- mlt-0.7.2.orig/src/mlt++/configure	2011-05-02 07:59:12.000000000 +0200
+++ mlt-0.7.2/src/mlt++/configure	2011-05-13 21:56:03.000000000 +0200
@@ -10,7 +10,7 @@
 		echo "CXXFLAGS+=-D__DARWIN__ -Wall -fPIC"
 		echo "LIBFLAGS=-dynamiclib -single_module"
 		;;
-	Linux|FreeBSD|NetBSD)
+	Linux|FreeBSD|NetBSD|GNU/kFreeBSD)
 		echo LIBSUF=.so
 		echo "CXXFLAGS+=-Wall $WARNINGS -fPIC -DPIC"
 		echo "LIBFLAGS=-shared"
