# Fix FTBFS on hurd.

diff -Naur mlt-0.7.6.orig/src/framework/mlt_repository.c mlt-0.7.6/src/framework/mlt_repository.c
--- mlt-0.7.6.orig/src/framework/mlt_repository.c	2011-10-31 18:26:04.000000000 +0100
+++ mlt-0.7.6/src/framework/mlt_repository.c	2011-11-05 11:27:53.898598558 +0100
@@ -39,6 +39,11 @@
 /** the default subdirectory of the datadir for holding presets */
 #define PRESETS_DIR "/presets"
 
+/** on hurd there is no PATH_MAX */
+#ifndef PATH_MAX
+#define PATH_MAX 1024
+#endif
+
 /** \brief Repository class
  *
  * The Repository is a collection of plugin modules and their services and service metadata.
