# Fix in / out points of the region's composite transition.

diff -Naur mlt-0.5.6.orig//src/modules/core/transition_region.c mlt-0.5.6/src/modules/core/transition_region.c
--- mlt-0.5.6.orig//src/modules/core/transition_region.c	2010-06-20 21:07:20.000000000 +0200
+++ mlt-0.5.6/src/modules/core/transition_region.c	2010-06-21 13:52:07.000000000 +0200
@@ -180,6 +180,9 @@
 			// We want to ensure that we don't get a wobble...
 			//mlt_properties_set_int( composite_properties, "distort", 1 );
 			mlt_properties_set_int( composite_properties, "progressive", 1 );
+			
+			mlt_properties_set_position( composite_properties, "in", mlt_transition_get_in( this ) );
+			mlt_properties_set_position( composite_properties, "out", mlt_transition_get_out( this ) );
 
 			// Pass all the composite. properties on the transition down
 			mlt_properties_pass( composite_properties, properties, "composite." );
@@ -193,6 +196,8 @@
 		// Pass all current properties down
 		mlt_properties composite_properties = MLT_TRANSITION_PROPERTIES( composite );
 		mlt_properties_pass( composite_properties, properties, "composite." );
+		mlt_properties_set_position( composite_properties, "in", mlt_transition_get_in( this ) );
+		mlt_properties_set_position( composite_properties, "out", mlt_transition_get_out( this ) );
 	}
 
 	// Create filters
