# Also detect hurd-i386 as Linux flavour.

diff -Naur mlt-0.7.6+git20111213.orig/configure mlt-0.7.6+git20111213/configure
--- mlt-0.7.6+git20111213.orig/configure	2011-12-13 19:28:26.639956316 +0100
+++ mlt-0.7.6+git20111213/configure	2011-12-13 19:40:41.200797535 +0100
@@ -88,7 +88,7 @@
 		echo "SHFLAGS=-dynamiclib"
 		echo "LDFLAGS+=`sdl-config --libs`"
 		;;
-		Linux|GNU/kFreeBSD)
+		Linux|GNU/kFreeBSD|GNU)
 		[ "$(uname -m)" = "x86_64" ] && echo "ARCH_X86_64=1" && echo "CFLAGS+=-DARCH_X86_64"
 		[ "$optimisations" = "true" ] &&
 			echo "OPTIMISATIONS+=-ffast-math"
