--- a/src/modules/sox/filter_sox.c
+++ b/src/modules/sox/filter_sox.c
@@ -181,6 +181,9 @@ static int create_effect( mlt_filter thi
 
 static int filter_get_audio( mlt_frame frame, int16_t **buffer, mlt_audio_format *format, int *frequency, int *channels, int *samples )
 {
+#if (ST_LIB_VERSION_CODE >= ST_LIB_VERSION(14,3,0))
+ 	SOX_SAMPLE_LOCALS;
+#endif
 	// Get the properties of the frame
 	mlt_properties properties = MLT_FRAME_PROPERTIES( frame );
 
