# Use the same targetos for hurd as for Linux.

diff -Naur mlt-0.7.6+git20111220.orig/src/mlt++/configure mlt-0.7.6+git20111220/src/mlt++/configure
--- mlt-0.7.6+git20111220.orig/src/mlt++/configure	2011-05-26 18:34:35.000000000 +0200
+++ mlt-0.7.6+git20111220/src/mlt++/configure	2011-12-20 19:33:25.195427389 +0100
@@ -10,7 +10,7 @@
 		echo "CXXFLAGS+=-D__DARWIN__ -Wall -fPIC"
 		echo "LIBFLAGS=-dynamiclib -single_module"
 		;;
-	Linux|FreeBSD|NetBSD|GNU/kFreeBSD)
+	Linux|FreeBSD|NetBSD|GNU/kFreeBSD|GNU)
 		echo LIBSUF=.so
 		echo "CXXFLAGS+=-Wall $WARNINGS -fPIC -DPIC"
 		echo "LIBFLAGS=-shared"
