--- a/Makefile
+++ b/Makefile
@@ -52,7 +52,7 @@ uninstall:
 		$(MAKE) DESTDIR=$(DESTDIR) -C $$subdir $@ || exit 1; \
 	done
 	rm -rf "$(DESTDIR)$(prefix)/include/mlt"
-	rm -rf "$(DESTDIR)$(prefix)/share/mlt"
+	rm -rf "$(DESTDIR)$(prefix)/lib/mlt"
 
 dist:
 	[ -d "mlt-$(version)" ] && rm -rf "mlt-$(version)" || echo
--- a/src/modules/westley/Makefile
+++ b/src/modules/westley/Makefile
@@ -30,7 +30,7 @@ clean:	
 
 install: all
 	install -m 755 $(TARGET) "$(DESTDIR)$(libdir)/mlt"
-	install -m 644 westley.dtd "$(DESTDIR)$(prefix)/share/mlt"
+	install -m 644 westley.dtd "$(DESTDIR)$(libdir)/mlt"
 
 ifneq ($(wildcard .depend),)
 include .depend
--- a/src/modules/fezzik/Makefile
+++ b/src/modules/fezzik/Makefile
@@ -28,8 +28,8 @@ clean:	
 
 install: all
 	install -m 755 $(TARGET) "$(DESTDIR)$(libdir)/mlt"
-	install -m 644 ../fezzik.dict "$(DESTDIR)$(prefix)/share/mlt"
-	install -m 644 ../fezzik.ini "$(DESTDIR)$(prefix)/share/mlt"
+	install -m 644 ../fezzik.dict "$(DESTDIR)$(libdir)/mlt"
+	install -m 644 ../fezzik.ini "$(DESTDIR)$(libdir)/mlt"
 
 ifneq ($(wildcard .depend),)
 include .depend
--- a/src/modules/avformat/Makefile
+++ b/src/modules/avformat/Makefile
@@ -69,13 +69,13 @@ clean:	
 
 install: all
 	install -m 755 $(TARGET) "$(DESTDIR)$(libdir)/mlt"
-	install -d "$(DESTDIR)$(prefix)/share/mlt/avformat"
-	install -m 644 producer_avformat.yml "$(DESTDIR)$(prefix)/share/mlt/avformat"
+	install -d "$(DESTDIR)$(libdir)/mlt/avformat"
+	install -m 644 producer_avformat.yml "$(DESTDIR)$(libdir)/mlt/avformat"
 
 uninstall:
 	rm "$(DESTDIR)$(libdir)/mlt/libmltavformat$(LIBSUF)" 2> /dev/null || true
 	rm "$(DESTDIR)$(libdir)/mlt/libmltffmpeg$(LIBSUF)" 2> /dev/null || true
-	rm -rf "$(DESTDIR)$(prefix)/share/mlt/avformat"
+	rm -rf "$(DESTDIR)$(libdir)/mlt/avformat"
 
 ifneq ($(wildcard .depend),)
 include .depend
--- a/src/modules/core/Makefile
+++ b/src/modules/core/Makefile
@@ -55,7 +55,7 @@ clean:	
 
 install: all
 	install -m 755 $(TARGET) "$(DESTDIR)$(libdir)/mlt"
-	install -m 644 ../data_fx.properties "$(DESTDIR)$(prefix)/share/mlt"
+	install -m 644 ../data_fx.properties "$(DESTDIR)$(libdir)/mlt"
 
 ifneq ($(wildcard .depend),)
 include .depend
--- a/src/modules/feeds/Makefile
+++ b/src/modules/feeds/Makefile
@@ -9,7 +9,7 @@ distclean:
 clean:
 
 install:	all
-	install -d $(DESTDIR)$(prefix)/share/mlt/feeds/PAL
-	install -d $(DESTDIR)$(prefix)/share/mlt/feeds/NTSC
-	install -m 644 PAL/*.* $(DESTDIR)$(prefix)/share/mlt/feeds/PAL
-	install -m 644 NTSC/*.* $(DESTDIR)$(prefix)/share/mlt/feeds/NTSC
+	install -d $(DESTDIR)$(libdir)/mlt/feeds/PAL
+	install -d $(DESTDIR)$(libdir)/mlt/feeds/NTSC
+	install -m 644 PAL/*.* $(DESTDIR)$(libdir)/mlt/feeds/PAL
+	install -m 644 NTSC/*.* $(DESTDIR)$(libdir)/mlt/feeds/NTSC
--- a/src/framework/Makefile
+++ b/src/framework/Makefile
@@ -92,15 +92,15 @@ install:
 	ln -sf $(TARGET) $(DESTDIR)$(libdir)/$(NAME)
 	install -d "$(DESTDIR)$(prefix)/include/mlt/framework"
 	install -m 644 $(INCS) "$(DESTDIR)$(prefix)/include/mlt/framework"
-	install -d "$(DESTDIR)$(prefix)/share/mlt"
-	install -m 644 metaschema.yaml "$(DESTDIR)$(prefix)/share/mlt/"
+	install -d "$(DESTDIR)$(libdir)/mlt"
+	install -m 644 metaschema.yaml "$(DESTDIR)$(libdir)/mlt/"
 
 uninstall:
 	rm -f "$(DESTDIR)$(libdir)/$(TARGET)"
 	rm -f "$(DESTDIR)$(libdir)/$(SONAME)"
 	rm -f "$(DESTDIR)$(libdir)/$(NAME)"
 	rm -rf "$(DESTDIR)$(prefix)/include/mlt/framework"
-	rm "$(DESTDIR)$(prefix)/share/mlt/metaschema.yaml"
+	rm "$(DESTDIR)$(libdir)/mlt/metaschema.yaml"
 
 ifneq ($(wildcard .depend),)
 include .depend
