--- a/src/modules/normalize/Makefile
+++ b/src/modules/normalize/Makefile
@@ -7,7 +7,7 @@ OBJS = factory.o \
 
 CFLAGS += -I../..
 
-LDFLAGS+=-L../../framework -lmlt
+LDFLAGS+=-L../../framework -lmlt -lm
 
 SRCS := $(OBJS:.o=.c)
 
--- a/src/modules/plus/Makefile
+++ b/src/modules/plus/Makefile
@@ -11,7 +11,7 @@ OBJS = factory.o \
 
 CFLAGS += -I../..
 
-LDFLAGS+=-L../../framework -lmlt
+LDFLAGS+=-L../../framework -lmlt -lm
 
 SRCS := $(OBJS:.o=.c)
 
--- a/src/modules/sdl/Makefile
+++ b/src/modules/sdl/Makefile
@@ -20,7 +20,7 @@ CFLAGS +=-I../.. `sdl-config --cflags`
 
 LDFLAGS +=`sdl-config --libs`
 
-LDFLAGS +=-L../../framework -lmlt
+LDFLAGS +=-L../../framework -lmlt -lpthread
 
 ifeq ($(WITH_SDL_IMAGE),1)
 OBJS += producer_sdl_image.o
--- a/src/modules/effectv/Makefile
+++ b/src/modules/effectv/Makefile
@@ -9,7 +9,7 @@ OBJS = factory.o \
 
 CFLAGS += -I../..
 
-LDFLAGS+=-L../../framework -lmlt
+LDFLAGS+=-L../../framework -lmlt -lm
 
 SRCS := $(OBJS:.o=.c)
 
--- a/src/modules/dv/Makefile
+++ b/src/modules/dv/Makefile
@@ -10,7 +10,7 @@ CFLAGS += `pkg-config --cflags libdv` -I
 
 LDFLAGS += `pkg-config --libs libdv`
 
-LDFLAGS+=-L../../framework -lmlt
+LDFLAGS+=-L../../framework -lmlt -lpthread
 
 SRCS := $(OBJS:.o=.c)
 
--- a/src/modules/avformat/Makefile
+++ b/src/modules/avformat/Makefile
@@ -29,7 +29,7 @@ CFLAGS+=-I../..
 
 LDFLAGS+=-L../../framework
 
-LDFLAGS+=-lavformat$(AVFORMAT_SUFFIX) -lavcodec$(AVFORMAT_SUFFIX) -lavutil$(AVFORMAT_SUFFIX) -lavdevice$(AVFORMAT_SUFFIX) $(EXTRA_LIBS) -lmlt
+LDFLAGS+=-lavformat$(AVFORMAT_SUFFIX) -lavcodec$(AVFORMAT_SUFFIX) -lavutil$(AVFORMAT_SUFFIX) -lavdevice$(AVFORMAT_SUFFIX) $(EXTRA_LIBS) -lmlt -lpthread
 
 ifdef SWSCALE
 	CFLAGS+=-DSWSCALE
--- a/src/modules/jackrack/Makefile
+++ b/src/modules/jackrack/Makefile
@@ -21,7 +21,7 @@ LDFLAGS += `pkg-config --libs jack`
 LDFLAGS += `xml2-config --libs`
 LDFLAGS += `pkg-config glib-2.0 --libs`
 
-LDFLAGS+=-L../../framework -lmlt
+LDFLAGS+=-L../../framework -lmlt -ldl -lm
 
 SRCS := $(OBJS:.o=.c)
 
--- a/src/modules/core/Makefile
+++ b/src/modules/core/Makefile
@@ -33,7 +33,7 @@ ASM_OBJS = 
 
 CFLAGS += -I../..
 
-LDFLAGS+=-L../../framework -lmlt
+LDFLAGS+=-L../../framework -lmlt -lm -lpthread
 
 SRCS := $(OBJS:.o=.c)
 
--- a/src/modules/motion_est/Makefile
+++ b/src/modules/motion_est/Makefile
@@ -12,7 +12,7 @@ OBJS = factory.o \
 
 CFLAGS += -I../.. 
 
-LDFLAGS+=-L../../framework -lmlt
+LDFLAGS+=-L../../framework -lmlt -lm
 
 SRCS := $(OBJS:.o=.c)
 
