--- a/src/modules/kino/kino_wrapper.cc
+++ b/src/modules/kino/kino_wrapper.cc
@@ -20,8 +20,8 @@
 
 #include "config.h"
 
-#include <string.h>
-#include <stdlib.h>
+#include <cstring>
+#include <cstdlib>
 
 #include "kino_wrapper.h"
 #include "filehandler.h"
--- a/src/modules/kino/avi.cc
+++ b/src/modules/kino/avi.cc
@@ -21,7 +21,7 @@
 
 // C++ includes
 
-#include <string>
+#include <cstring>
 #include <iostream>
 #include <iomanip>
 
--- a/src/modules/kino/filehandler.cc
+++ b/src/modules/kino/filehandler.cc
@@ -23,7 +23,7 @@
 #include <framework/mlt_frame.h>
 }
 
-#include <string>
+#include <cstring>
 #include <iostream>
 #include <sstream>
 #include <iomanip>
--- a/configure
+++ b/configure
@@ -70,7 +70,7 @@
 		Linux)
 		echo "OPTIMISATIONS+=-ffast-math"
 		echo "CFLAGS+=-pthread"
-		echo "LDFLAGS+= -Wl,--no-undefined -Wl,--as-needed"
+		echo "LDFLAGS+= -Wl,--no-undefined -Wl,--as-needed -lm"
 		echo "SHFLAGS=-shared"
 		echo "LIBDL=-ldl"
 		echo "RDYNAMIC=-rdynamic"
